pkg/apiserver/rest/rest_server.go

```diff
@@ -117,6 +118,8 @@ func (s *restServer) Run(ctx context.Context) error {
 		leaderelection.RunOrDie(ctx, *l)
 	}()
 
+	go velasync.Start(s.dataStore)
```

Comment:
Multiple synchronizers work when multiple instances run. so there is a leader election in line 118.

Reply:
yes, thanks for reminder